### PR TITLE
sage/interacting-with-smart-contract-on-celo-using-web3py

### PR DESCRIPTION
**Pull request by bobeu**
Deleted previous PR due to file conflicts. All issues are resolved.

### CODE REVIEW

File: blog/04-22-2023-interacting-with-smart-contract-on-celo-using-web3py/interacting-with-smart-contract-on-celo-using-web3py.md [hunk 1]
```diff
@@ -0,0 +1,1203 @@
+---
+title: Interacting with smart contract on celo using web3py
+description: This tutorial teaches you how to set up and use Web3py - A python-based web3 library
+authors:
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [celo, solidity, smartcontract]
```
Comment (yafiabiyyu) [inline blog/04-22-2023-interacting-with-smart-contract-on-celo-using-web3py/interacting-with-smart-contract-on-celo-using-web3py.md:9]:
```suggestion
tags: ["celosage", "intermediate", "smartcontract", "solidity"]
```

Comment (yafiabiyyu) [reply blog/04-22-2023-interacting-with-smart-contract-on-celo-using-web3py/interacting-with-smart-contract-on-celo-using-web3py.md:9]: Make sure that the tags you use in markdown are the same as the tags you use in users.tsx

Comment (yafiabiyyu) [inline blog/04-22-2023-interacting-with-smart-contract-on-celo-using-web3py/interacting-with-smart-contract-on-celo-using-web3py.md:9]:
```suggestion
tags: ['celosage', 'intermediate', 'smartcontract', 'solidity']
```

**Review by developerkunal — APPROVED** [review]
Approved

**Pull request merged**